### PR TITLE
expression: implement vectorized evaluation for 'builtinDurationAnyValueSig'

### PR DESCRIPTION
**Pull request by tsthght**
### What problem does this PR solve? 
Implement vectorized evaluation for builtinDurationAnyValueSig, for #12176

### What is changed and how it works?
```
BenchmarkVectorizedBuiltinMiscellaneousCasesFunc/builtinDurationAnyValueSig-VecBuiltinFunc-8        20000000	       115 ns/op	       0 B/op	       0 allocs/op
BenchmarkVectorizedBuiltinMiscellaneousCasesFunc/builtinDurationAnyValueSig-NonVecBuiltinFunc-8       100000	     20525 ns/op	       0 B/op	       0 allocs/op
```

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by tsthght** [comment]
./rebuild

File: expression/builtin_miscellaneous_vec.go [hunk 1]
```diff
@@ -161,11 +161,14 @@ func (b *builtinLockSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) er
 }
 
 func (b *builtinDurationAnyValueSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinDurationAnyValueSig) vecEvalDuration(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	if err := b.args[0].VecEvalDuration(b.ctx, input, result); err != nil {
```
Comment (SunRunAway) [inline expression/builtin_miscellaneous_vec.go]: use ` b.args[0].VecEvalDuration(b.ctx, input, result)` for shortly?

Comment (tsthght) [reply expression/builtin_miscellaneous_vec.go]: ok~

**Comment by qw4990** [comment]
@tsthght Please format the file `builtin_miscellaneous_vec_test` to pass the check.

**Comment by qw4990** [comment]
@tsthght Please format `expression/builtin_miscellaneous_vec_test.go` 🤣

**Pull request merged**